### ISSUE TITLE
Duplicate Question Handling

### ISSUE THREAD

**Issue by sjohnson620** [nthopinion/covid19#46]
Need to look at ways to manage duplicate questions that may have slight differences (incorrect spelling, different sentence structure, translation error, etc.) . Will try to think through solutions- if coder has thoughts please pick up and run with it.

**Comment by sjohnson620** [comment]
@aviskarkc10 This is a result of #43 

Was this your thought?

**Comment by aviskarkc10** [comment]
Yes. we should probably create an api which will be called when user starts typing the ask question form. The api will return a list of questions that could be similar to the one the user is typing.

We can do this through cosine similarity between sentences. Maybe there's a person who's done NLP work in the slack. Especially the `ai-ml-rl` channel?

**Comment by nnation** [comment]
@aviskarkc10 do you know if anyone took this up?

**Comment by aviskarkc10** [comment]
No. I think this issue is on hold. This is a difficult problem to solve and we haven't had anyone pick it up or even discuss how to acheive.
